### PR TITLE
BI-10194 Map officer type code from oracle to chs format

### PR DESCRIPTION
**Pull request by mattch1**

### CODE REVIEW

File: src/test/java/uk/gov/companieshouse/confirmationstatementapi/model/mapping/OfficerIdentificationTypeMapperTest.java [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+    }
+
+    @Test
+    void testOtherCorporateBodyOrFirmMapping() {
+        String chsType = OfficerIdentificationTypeMapper.mapIdentificationTypeToChs("G");
+        assertEquals("other-corporate-body-or-firm", chsType);
+    }
+
+    @Test
+    void testUnrecognizedMapping() {
```
Comment (markpit) [inline src/test/java/uk/gov/companieshouse/confirmationstatementapi/model/mapping/OfficerIdentificationTypeMapperTest.java:35]: add test for a null string and a test for blank string

File: src/test/java/uk/gov/companieshouse/confirmationstatementapi/model/mapping/OfficerIdentificationTypeMapperTest.java [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+    @Test
+    void testOtherCorporateBodyOrFirmMapping() {
+        String chsType = OfficerIdentificationTypeMapper.mapIdentificationTypeToChs("G");
+        assertEquals("other-corporate-body-or-firm", chsType);
+    }
+
+    @ParameterizedTest
+    @ValueSource(strings = {"Z", " ", ""})
+    void testNullUnrecognizedMappings(String type) {
+        System.out.println("TYPE: " + type);
```
Comment (markpit) [inline src/test/java/uk/gov/companieshouse/confirmationstatementapi/model/mapping/OfficerIdentificationTypeMapperTest.java:40]: the System.out.println will need to be removed

**Review by markpit — APPROVED** [review]
